test/Table_Tests/src/Database/Common_Spec.enso

```diff
@@ -321,33 +321,33 @@ spec prefix connection pending=Nothing =
             table.order_by (Sort_Column_Selector.By_Name [Sort_Column.Name order_column])
 
         Test.specify "should allow counting group sizes and elements" <|
-            aggregates = [Count Nothing, Count_Not_Nothing "price", Count_Nothing "price"]
+            aggregates = [Count "count", Count_Not_Nothing "price" "count not nothing price", Count_Nothing "price" "count nothing price"]
```

Comment:
Please add a comment why this change happened (presumably Redshift dealing weirdly with case?). Otherwise when looking at this in the future we won't know why the names were changed this way, may try to change them back and lose time experimenting.

Besides, I think we may need to think of a more robust approach for fixing the rest of the tests - maybe - just maybe - one solution could be to know if a database uses case insensitive columns and for such a backend do the same for our underlying table - i.e. `redshift_table.at "MIXED_case_NaMe"` will find the `mixed_case_name` column. Although I'm not sure if that's a good solution, just tossing ideas around. For now the above looks good, but please add a comment so that we know the rationale behind that.